src/Utilities/Compiler/Options/SymbolNamesWithValueOption.cs

```diff
@@ -19,11 +19,11 @@ internal sealed class SymbolNamesWithValueOption<TValue> : IEquatable<SymbolName
         internal const SymbolKind AllKinds = SymbolKind.ErrorType;
         internal const char WildcardChar = '*';
 
-        public static readonly SymbolNamesWithValueOption<TValue> Empty = new SymbolNamesWithValueOption<TValue>();
-        internal static KeyValuePair<string, TValue> NoWildcardMatch => default;
+        public static readonly SymbolNamesWithValueOption<TValue> Empty = new();
+        internal static KeyValuePair<string, TValue?> NoWildcardMatch => default;
```

Comment:
This had no nullable warning before. Is it one of the known pitfalls to nullable feature?

[Minimal repro](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxAgrgOwD4GJPoA2hEwhcABHJqeQLABQjAAgEwCMLADBc+wCwBuRowDScAJ4A1CIXRwAChACWCADzLMMJL3ZcAfBQDWANwoBeCgBM4AMwhEYwhnwCcAClMA6GXLheAFQB7AGUYBE0Ac3cAShjBIA===)